### PR TITLE
Clarify CONTRIBUTING guidelines

### PR DESCRIPTION
**Pull request by chriszarate**
## Description

Update `CONTRIBUTING` to make the release process clear, especially:

- who is responsible for what
- when releases to NPM happen
- how to avoid accidentally releasing something

This may reflect a _complete_ misreading / misunderstanding on my part. If that's the case, let me know and I will update it. :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -52,11 +52,12 @@ Our release flow for VIP CLI follows this pattern:
 Changelogs allow customers to keep up with all the changes happening across our VIP Platform. Changelogs for VIP CLI are posted to the V[VIP Cloud Changelog P2](https://wpvipchangelog.wordpress.com/), along with the repository’s `README.md`.
 
 ### Release Schedule
-We deploy changes from our `develop` branch to the `master` branch on Tuesdays from 4AM to 5PM Pacific Time.
 
-Changes that are not ready to be publicly released should stay in the `develop` branch to avoid conflicts with any urgent fixes we may need to roll out. Fixes for urgent or breaking bugs may be merged straight to the `master` branch and released publicly via npm at any time outside of our release schedule.
+The VIP Customer Experience squad conducts releases on Tuesdays from 4AM to 5PM Pacific Time (US). Releases are regular but do not happen every Tuesday. Please coordinate with the VIP Customer Experience squad to determine the next scheduled release date.
```
Comment (mjangda) [inline CONTRIBUTING.md]: I would note here that other teams can push releases as well with support from Patisserie (if needed). We don't want a single team to be a blocker for getting changes rolled out.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -52,11 +52,12 @@ Our release flow for VIP CLI follows this pattern:
 Changelogs allow customers to keep up with all the changes happening across our VIP Platform. Changelogs for VIP CLI are posted to the V[VIP Cloud Changelog P2](https://wpvipchangelog.wordpress.com/), along with the repository’s `README.md`.
```
Comment (chriszarate) [inline CONTRIBUTING.md]:
```suggestion
Changelogs allow customers to keep up with all the changes happening across our VIP Platform. Changelogs for VIP CLI are posted to the [VIP Cloud Changelog P2](https://wpvipchangelog.wordpress.com/), along with the repository’s `README.md`.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (chriszarate) [reply CONTRIBUTING.md]:
```suggestion
The VIP Customer Experience squad conducts releases on Tuesdays from 4AM to 5PM Pacific Time (US). Releases are regular but do not happen every Tuesday. Please coordinate with the VIP Customer Experience squad if you'd like to lead your own release.
```

**Review by pandah3 — APPROVED** [review]
Awesome, thank you 🙏 !!

**Pull request merged**